fc-schema-cli does not compile on java 11

```
[INFO] 
[INFO] --- kotlin-maven-plugin:1.3.61:kapt (kapt) @ factcast-schema-registry-cli ---
[WARNING] 'tools.jar' was not found, kapt may work unreliably
[INFO] Applied plugin: 'all-open'
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jetbrains.kotlin.kapt3.base.javac.KaptJavaFileManager (file:/home/doc/.m2/repository/org/jetbrains/kotlin/kotlin-annotation-processing-maven/1.3.61/kotlin-annotation-processing-maven-1.3.61.jar) to method com.sun.tools.javac.file.BaseFileManager.handleOption(com.sun.tools.javac.main.Option,java.lang.String)
WARNING: Please consider reporting this to the maintainers of org.jetbrains.kotlin.kapt3.base.javac.KaptJavaFileManager
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Note: Writing native-image.properties file to destination: META-INF/native-image/org.factcast.schema.registry/cli/native-image.properties
[INFO] Note: Writing reflection-config.json file to destination: META-INF/native-image/org.factcast.schema.registry/cli/reflection-config.json
[INFO] Note: Creating bean classes for 80 type elements
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ factcast-schema-registry-cli ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 413 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ factcast-schema-registry-cli ---
[INFO] Changes detected - recompiling the module!
[INFO] 
[INFO] --- kotlin-maven-plugin:1.3.61:compile (compile) @ factcast-schema-registry-cli ---
[INFO] Applied plugin: 'all-open'
[ERROR] /home/doc/work/workspace/fc/factcast/factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt: (20, 34) Symbol is declared in module 'java.base' which does not export package 'sun.net.www.protocol.file'
[ERROR] /home/doc/work/workspace/fc/factcast/factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt: (78, 16) Symbol is declared in module 'java.base' which does not export package 'sun.net.www.protocol.file'
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] factcast 0.1.5-SNAPSHOT ............................ SUCCESS [  9.577 s]
[INFO] factcast-bom ....................................... SUCCESS [  0.070 s]
[INFO] factcast-schema-registry-cli ....................... FAILURE [ 19.397 s]
[INFO] factcast-site ...................................... SKIPPED
[INFO] factcast-internal-dep .............................. SKIPPED
[INFO] factcast-internal-dep-boot ......................... SKIPPED
[INFO] factcast-core ...................................... SKIPPED
[INFO] factcast-grpc-api .................................. SKIPPED
[INFO] factcast-client-grpc ............................... SKIPPED
[INFO] factcast-store-pgsql-rds ........................... SKIPPED
[INFO] factcast-store-test ................................ SKIPPED
[INFO] factcast-store-pgsql ............................... SKIPPED
[INFO] factcast-server-grpc ............................... SKIPPED
[INFO] factcast-spring-boot-autoconfigure ................. SKIPPED
[INFO] factcast-client-grpc-cli ........................... SKIPPED
[INFO] factcast-docker .................................... SKIPPED
[INFO] factcast-examples .................................. SKIPPED
[INFO] factcast-example-client-basicauth .................. SKIPPED
[INFO] factcast-example-client-spring-boot2 ............... SKIPPED
[INFO] factcast-example-server ............................ SKIPPED
[INFO] factcast-example-server-basicauth .................. SKIPPED
[INFO] factcast-example-tls-client ........................ SKIPPED
[INFO] factcast-example-tls-server ........................ SKIPPED
[INFO] factcast-example-schema-registry ................... SKIPPED
[INFO] factcast-integration-full-tls 0.1.5-SNAPSHOT ....... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 30.179 s
[INFO] Finished at: 2020-03-02T08:36:18+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.jetbrains.kotlin:kotlin-maven-plugin:1.3.61:compile (compile) on project factcast-schema-registry-cli: Compilation failure: Compilation failure: 
[ERROR] /home/doc/work/workspace/fc/factcast/factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt:[20,34] Symbol is declared in module 'java.base' which does not export package 'sun.net.www.protocol.file'
[ERROR] /home/doc/work/workspace/fc/factcast/factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt:[78,16] Symbol is declared in module 'java.base' which does not export package 'sun.net.www.protocol.file'
[ERROR] -> [Help 1]
```

--- a/factcast-schema-registry-cli/pom.xml
+++ b/factcast-schema-registry-cli/pom.xml
@@ -151,6 +151,11 @@
       <artifactId>jackson-core</artifactId>
       <version>2.10.2</version>
     </dependency>
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.2</version>
+    </dependency>
   </dependencies>
   <build>
     <plugins>
@@ -223,7 +228,8 @@
                 <argument>target/fc-schema-cli.jar</argument>
                 <argument>build</argument>
                 <argument>--base-path</argument>
-                <argument>../factcast-examples/factcast-example-schema-registry/src/main/resources</argument>
+                <argument>../factcast-examples/factcast-example-schema-registry/src/main/resources
+                </argument>
               </arguments>
             </configuration>
           </execution>

--- a/factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt
+++ b/factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt
@@ -17,7 +17,6 @@ package org.factcast.schema.registry.cli.fs
 
 import com.github.fge.jackson.JsonLoader
 import org.apache.commons.io.FileUtils
-import sun.net.www.protocol.file.FileURLConnection
 import java.io.File
 import java.io.IOException
 import java.net.JarURLConnection
@@ -31,15 +30,15 @@ import kotlin.streams.toList
 @Singleton
 class FileSystemServiceImpl : FileSystemService {
     override fun exists(path: Path) =
-        Files.exists(path)
+            Files.exists(path)
 
     override fun listDirectories(path: Path) =
-        list(path)
-            .filter { x -> Files.isDirectory(x) }
+            list(path)
+                    .filter { x -> Files.isDirectory(x) }
 
     override fun listFiles(path: Path) =
-        list(path)
-            .filter { x -> Files.isRegularFile(x) }
+            list(path)
+                    .filter { x -> Files.isRegularFile(x) }
 
     override fun ensureDirectories(outputPath: Path) {
         Files.createDirectories(outputPath)
@@ -50,7 +49,7 @@ class FileSystemServiceImpl : FileSystemService {
     }
 
     override fun readToString(filePath: File): String =
-        FileUtils.readFileToString(filePath, Charset.defaultCharset())
+            FileUtils.readFileToString(filePath, Charset.defaultCharset())
 
     override fun copyFile(from: File, to: File) {
         FileUtils.copyFile(from, to)
@@ -63,7 +62,7 @@ class FileSystemServiceImpl : FileSystemService {
     }
 
     override fun deleteDirectory(path: Path) =
-        FileUtils.deleteDirectory(path.toFile())
+            FileUtils.deleteDirectory(path.toFile())
 
     override fun readToBytes(file: Path): ByteArray {
         return Files.readAllBytes(file)
@@ -71,13 +70,15 @@ class FileSystemServiceImpl : FileSystemService {
 
     override fun copyFromClasspath(source: String, target: Path) {
         val url = javaClass.classLoader.getResource(source)
-            ?: throw IllegalArgumentException("didnt found '$source' on classpath")
+                ?: throw IllegalArgumentException("didnt found '$source' on classpath")
 
-        return when (val urlConnection: URLConnection = url.openConnection()) {
-            is JarURLConnection -> copyJarResourcesRecursively(target.toFile(), urlConnection)
-            is FileURLConnection -> FileUtils.copyDirectory(File(url.path), target.toFile())
-            else -> throw IllegalStateException("not supported")
-        }
+        val urlConnection: URLConnection = url.openConnection()
+        val toString = url.toString()
+        if (toString.startsWith("file:")) {
+            FileUtils.copyDirectory(File(url.path), target.toFile())
+        } else if (toString.startsWith("jar:")) {
+                copyJarResourcesRecursively(target.toFile(), urlConnection as JarURLConnection)
+            } else throw IllegalStateException("not supported")
     }
 
     override fun copyDirectory(from: Path, to: Path) {
@@ -85,7 +86,7 @@ class FileSystemServiceImpl : FileSystemService {
     }
 
     private fun list(path: Path) =
-        Files.list(path).toList()
+            Files.list(path).toList()
 
     private fun copyJarResourcesRecursively(destination: File, jarConnection: JarURLConnection) {
         val jarFile = jarConnection.jarFile

